components/typography/Link.tsx

```diff
@@ -0,0 +1,13 @@
+import styles from "./Link.module.css";
+import { ReactNode } from "react";
+
+type LinkProps = { 
+    children: ReactNode,
+    className?: string, 
+};
+  
+  const Link: React.FC<LinkProps> = ({ children, className }) => {
+    return <a className={`${styles.link} ${className}`}>{children}</a>   
```

Comment:
if you pass a `href="somewebsite.nl"` it will not work the moment
nice way to do this is to use props spreading

```ts
type LinkProps extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
  // no additional types are needed
}

const Link = ({ children, className, ...props } : LinkProps) => {
    return <a {...props} className={`${styles.link} ${className}`}>{children}</a>   
}
```

in order to do this, you will need some additional TypeScript definitions